lib/improver/tests/wxcode/wxcode/test_wxcode_utilities.py

```diff
@@ -271,6 +271,16 @@ def test_metadata_set(self):
         self.assertEqual(result.attributes['weather_code_meaning'],
                          self.wxmeaning)
 
+    def test_metadata_removes_bounds(self):
+        """Test that the metadata removes bounds from cube."""
+        cube = self.cube.copy()
+        p1 = cube.coord('time').points[0]
+        p2 = p1 - 60*60
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable name "p2" doesn't conform to snake_case naming style](https://app.codacy.com/app/metoppv_tech/improver/pullRequest?prid=4535748)